model/userModel.js

```diff
@@ -41,6 +41,20 @@ const userSchema = Mongoose.Schema({
         type: String,
         default: '',
         trim: true
+    },
+    notifications: {
+        type: [new Mongoose.Schema({
```

Comment:
Not obligatory but this could be make into a real schema like UserNotificationSchema, like we did for SimpleUserSchema.

Reply:
`SimpleUserSchema` was created because it is used at multiple places, and as such to avoid changes at two different places if a change is necessary. I would prefer to keep it here, to have it all at one place, gives a clearer image of the schema in question. If the need arises to have it at more places or if it gets unnecessary complicated it will be moved to a separate file. Until then, best keep it as is.